cloud/ociutil/ociutil.go

```diff
@@ -80,6 +83,28 @@ func AwaitLBWorkRequest(ctx context.Context, networkLoadBalancerClient nlb.Netwo
 	return wr, err
 }
 
+// AwaitLbsLBWorkRequest waits for the LBaaS work request to either succeed, fail. See k8s.io/apimachinery/pkg/util/wait
+func AwaitLbsLBWorkRequest(ctx context.Context, loadBalancerClient lbs.LoadBalancerServiceClient, workRequestId *string) (*loadbalancer.WorkRequest, error) {
```

Comment:
AwaitLBWorkRequest

Reply:
Since NLB uses this function name, I'll update it in both related files.